audit/src/main/java/com/ritense/audit/service/impl/AuditRetentionServiceImpl.java

```diff
@@ -33,6 +34,9 @@ public AuditRetentionServiceImpl(AuditService auditService, long retention) {
 
     @Override
     @Scheduled(cron = "${scheduling.job.cron.cleanupAuditEvents:-}")
+    @SchedulerLock(
```

Comment:
The 'ReminderService' and the new Exact plugin also use a cron job. Could we also add the @SchedulerLock there?